src/z3c/autoinclude/utils.py

```diff
@@ -206,3 +212,29 @@ def find_packages(where='.', exclude=()):
 
         out = [item for item in out if not fnmatchcase(item, pat)]
     return out
+
+
+def create_report(info):
+    """Create a report with a list of auto included zcml."""
+    if not info:
+        # Return a comment.  Maybe someone wants to automatically include this
+        # in a zcml file, so make it a proper xml comment.
+        return ["<!-- No zcml files found to include. -->"]
+    report = []
+    # Try to report meta.zcml first.
+    filenames = list(info)
+    meta = "meta.zcml"
+    if meta in filenames:
+        filenames.pop(filenames.index(meta))
```

Comment:
There's a shorter spelling for this: `filenames.remove(meta)`.

Reply:
Of course. Done.